gpytorch/lazy/mul_lazy_variable.py

```diff
@@ -0,0 +1,157 @@
+from .lazy_variable import LazyVariable
+from ..posterior import DefaultPosteriorStrategy
+import gpytorch
+import torch
+from torch.autograd import Variable
+
+
+class MulLazyVariable(LazyVariable):
+    def __init__(self, *lazy_vars):
+        if not all([isinstance(lazy_var, LazyVariable) for lazy_var in lazy_vars]):
+            raise RuntimeError('All arguments of a SumLazyVariable should be lazy variables')
```

Comment:
Should be "All arguments of a MulLazyVariable"

Reply:
Let's add a Trello card for support of add and mul of LazyVariables with standard Torch Variables.